pkg/secrets/backup_test.go

```
@@ -266,6 +264,52 @@ var _ = Describe("HandleRegistryAuthSecret (backup path: operatorConfigNamespace
+	)
+
+	var (
+		ctx    context.Context
+		scheme *runtime.Scheme
+		log    = zap.New(zap.UseDevMode(true)).WithName("SecretsTest")
+	)
+
+	BeforeEach(func() {
+		ctx = context.Background()
+		scheme = buildScheme()
+		os.Setenv(infrastructure.WatchNamespaceEnvVar, operatorNS)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Setenv` is not checked

(errcheck)